src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: $primary-background-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-family: inherit;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-family: inherit;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    margin-top: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__reviews-text {
+    font-family: inherit;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+  &__price-item {
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    font-family: inherit;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price {
+    font-family: inherit;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent-color;
+  }
+
+  &__button {
+    margin-top: 16px;
+    font-family: inherit;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    padding: 11px 0;
+    text-align: center;
+    text-transform: uppercase;
+    color: $primary-background-color;
+    background: $header-secondary-color;
+    border-radius: 5px;
+    text-decoration: none;
+    border: 1px solid transparent;
+  }
+
+  &__button:hover {
+    color: $header-secondary-color;
+    background: $primary-background-color;
+    border: 1px solid $header-secondary-color;
```

Comment:
add this border directly to `&__button` and you don't need to change border on hover